api/src/main/java/uk/gov/dhsc/htbhf/claimant/scheduler/AutowiringSpringBeanJobFactory.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.dhsc.htbhf.claimant.scheduler;
+
+import org.quartz.spi.TriggerFiredBundle;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.scheduling.quartz.SpringBeanJobFactory;
+
+/**
+ * {@link org.quartz.spi.JobFactory} implementation that is spring aware, allowing {@link org.quartz.Job} instances
+ * to autowire spring bean dependencies.
+ * {@see <a href="https://dzone.com/articles/spring-and-quartz-integration-that-works-together/> here </a>} for a more complex example
+ * which gives the job access to {@link org.quartz.JobDataMap}s and the {@link org.quartz.SchedulerContext}. (not need for our job)
+ */
+public class AutowiringSpringBeanJobFactory extends SpringBeanJobFactory implements ApplicationContextAware {
+
+    private ApplicationContext applicationContext;
+
+    @Override
+    public void setApplicationContext(final ApplicationContext applicationContext) {
```

Comment:
We don't need final on this parameter (also on method below too).

Reply:
Removed.